src/components/Card.js

```diff
@@ -1,9 +1,18 @@
 export default class Card {
-  constructor({ cardData, handleCardClick }, cardSelector) {
+  constructor(
+    { cardData, handleCardClick, handleDeleteSubmit, handleCardLike },
+    cardSelector
+  ) {
     this._name = cardData.name;
     this._link = cardData.link;
-    this._cardSelector = cardSelector;
+    this._cardID = cardData._id;
+    this._userID = cardData._userID;
+    this._cardLike = cardData.likes;
```

Comment:
Please rename this variable to `_cardLikes`, because there are many likes, not one